src/ex6/client/src/components/main-window.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect, useCallback } from "react";
+import { AddTodoForm } from "./add-todo-form.js";
+import { NoTodosPlaceholder } from "./no-todos-placeholder.js";
+import { TodosList } from "./todos-list.js";
+import { Footer } from "./footer.js";
+import { getItems } from "../item_client.js";
+
+export function MainWindow() {
```

Comment:
consider renaming to a more specific name (ex: TodosCard/ TodosContainer.. etc)

Reply:
done